apps/vscode/command_client/command_client.py

```diff
@@ -319,16 +301,16 @@ def vscode_with_plugin(
             arg5,
         )
 
-    def vscode_with_plugin_and_wait(
+    def run_rpc_command_and_wait(
```

Comment:
yeah this name is not bad.  alternatively `run_ide_command...` or `run_editor_command...`.  But `rpc` seems reasonable

Reply:
I like `run_rpc_command`, it's appropriately general.

Reply:
I already started using this for something that's technically not an editor, so agree rpc is better